hack/install_kustomize.sh

```diff
@@ -44,12 +44,13 @@ elif [[ "$OSTYPE" == darwin* ]]; then
   opsys=darwin
 fi
 
-curl -s https://api.github.com/repos/kubernetes-sigs/kustomize/releases |\
+curl -s $release_url |\
   grep browser_download |\
   grep $opsys |\
```

Comment:
We should address the arch type here as well. We shouldn't download arm file  on amd64 system.

Reply:
I have consolidated the grep statements to address this.
I was considering adding the ability to download arm on arm systems using `dpkg --print-architecture` but not completely sure how the output differ is on arm 32-bit; AFAIK arm 64-bit will always be `arm64`

Reply:
`dpkg` is not available on all system I believe. We can use amd64 here and make a note on the document page for arm versions.